spec/units/commands/internal/chdir_spec.rb

```diff
@@ -6,14 +6,14 @@
 
   describe '#execute' do
     it 'returns true for correct directories' do
-      env = stub(:[]= => true, puts_error: true)
+      env = double("Environment", :[]= => true, puts_error: true)
```

Comment:
I know, I recently started using the habbit of using double-quotes myself. But it seems hard getting that out of my muscle memory. Sorry!